src/Container.tsx

```diff
@@ -0,0 +1,35 @@
+/* @jsx jsx */
+
+import { jsx } from '@emotion/core';
+import { FunctionComponent } from 'react';
+
+import { Box, IBox } from './Box';
+import { useTheme } from './theme';
+
+export interface IContainer extends IBox {}
+
+export const Container: FunctionComponent<IContainer> = props => {
```

Comment:
Yes, that's exactly this case. Having predefined styles on Box turned out to be quite annoying, as I had to override quite often and I couldn't rely on inheritance.

Reply:
Hmm, want to think about this for a sec - could you share an example of the predefined styles on box causing issues?

Reply:
It caused issues in combination with different UI-Kit's components that had their own styling, i.e.

```tsx
<Heading backgroundColor="red">
    Title
    <Box as="span" fontSize="12px">foo</Box>
</Heading>
```

Actual view:
![image](https://user-images.githubusercontent.com/9273484/49954562-0b092900-ff01-11e8-9bb7-59092ca8c4cd.png)

Reply:
Ah, I feel like box should not make many assumptions, so we might want to fix something here. Perhaps `box` should not have a theme entry? IE no default background, color, etc.

Reply:
Is the thought that container replaces the idea of canvas as a theme zone?

Reply:
Removing default colors on box makes sense and they are removed in this PR.
Introducing a separate component (such as container) is discussable, but IMHO it's worth to have such a component.
It's just easier to develop end-application.
Instead of writing:
```tsx
<Box backgroundColor={someColor} color={someColor}>
  foo
</Box>
```
you just write
```tsx
<Container>foo
</Container>
```
or
```tsx
<Container as={Flex}>

</Container>
```
if you want to apply base color styles to some other component.

It also let's you opt-out/override parent's component styles, i.e.

```tsx
<Box backgroundColor="red">
  I have red backgrond
  <Container>I black background, because I use dark theme and I don't have backgroundColor attribute
  </Container>
</Box>
```

Reply:
But if box component doesn't set any colors by default, can't you effectively use it the same way that you are using container above?

Reply:
The thing is that at the moment (before this PR) it adds some colors. I reverted that change in this PR and added this container as a complementary component to still have some default colors from the theme.
Makes sense?